src/Engine/Mandarin/Mandarin.cpp

```diff
@@ -177,7 +177,7 @@ const BPMF BPMF::FromHanyuPinyin(const std::string& str) {
   }
 
   // then we try ZH, CH, SH, R, Z, C, S (in that order)
-  if (0) {
+  if (false) {  // NOLINT(readability-simplify-boolean-expr)
```

Comment:
All the `if (false)` occurences can be removed. Same below.

Reply:
fixed in https://github.com/openvanilla/fcitx5-mcbopomofo/pull/84/commits/fe162788bb678085edc195ab09e40013eea0f95e